lib/version_utils.pm

```diff
@@ -113,6 +114,13 @@ sub is_jeos {
     return get_var('FLAVOR', '') =~ /^JeOS/;
 }
 
+=head2 is_community_jeos
+
+Returns true if called on community jeos images
+=cut
+sub is_community_jeos {
+    return check_var('FLAVOR', 'JeOS-for-AArch64');
```

Comment:
Just found that it's also `JeOS-for-RPi`: https://openqa.opensuse.org/tests/1657483#settings

As opposed to `JeOS-for-RaspberryPi` which is the other JeOS... How confusing.